README.md

```diff
@@ -0,0 +1,36 @@
+# Percona Distribution for PostgreSQL documentation.
+
+Welcome to Percona Distribution for PostgreSQL documentation!
+
+## Overview
+
+Percona Distribution for PostgreSQL is a collection of tools to assist you in managing your PostgreSQL database system. It includes the upstream version of PostgreSQL and a selection of extensions that enable solving essential practical tasks efficiently.
+ 
+This repository contains the source files for [Percona Distribution for PostgreSQL documentation](https://www.percona.com/doc/postgresql/11/index.html). The documentation is written in [reStructured text markup language](https://docutils.sourceforge.io/rst.html) and is created using [Sphinx Python Documentation Generator](https://www.sphinx-doc.org/en/master/). 
+
+## Installation
+
+Percona Distribution for PostgreSQL is installed from Percona repositories. Follow the [installation instructions](https://www.percona.com/doc/postgresql/11/installing.html) in our documentation.
+
+## Submitting bug reports
+
+If you find a bug in Percona Distribution for PostgreSQL, you can submit a report to the project's [JIRA issue tracker](https://jira.percona.com/projects/DISTPG).
+
+As a general rule of thumb, please try to create bug reports that are:
+
+- Reproducible. Include steps to reproduce the problem.
+- Specific. Include as much detail as possible: which version, what environment, etc.
+- Unique. Do not duplicate existing tickets.
+- Scoped to a Single Bug. One bug per report.
+
+## Contributing
+
+We welcome all contributions and are always looking for new members that are as dedicated to serving the community as we are. You can reach out to us using our [forums](https://forums.percona.com/c/postgresql/25) and [Jira issue tracker](https://jira.percona.com/projects/DISTPG/issues/DISTPG-16?filter=allopenissues). 
+
+Since we don't maintain code of Percona Distribution for PostgreSQL, please submit your code contributions upstream.
```

Comment:
I would remove that sentence from this README, let's focus on instructions on how to contribute to this documentation.

Reply:
Removed